plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -110,7 +148,7 @@ private void assertPropertiesDefined(Element element) {
 
   private void assertPropertyDefined(String name) {
```

Comment:
I'm not sure "externalized" is the right word here. This test is checking that a property used in plugin.xml is present in plugin.properties with no typos or misspellings. build.properties isn't involved in this one.